netci.groovy

```diff
@@ -124,7 +124,7 @@ def CreateXPlatBuildTask = { isPR, buildType, staticBuild, machine, platform, co
     def staticFlag = staticBuild ? "--static" : ""
     def swbCheckFlag = (platform == "linux" && buildType == "debug" && !staticBuild) ? "--wb-check" : "";
     def icuFlag = (platform == "osx" ? "--icu=/usr/local/opt/icu4c/include" : "")
-    def compilerPaths = (platform == "osx") ? "" : "--cxx=/usr/bin/clang++-3.8 --cc=/usr/bin/clang-3.8"
+    def compilerPaths = (platform == "osx") ? "" : "--cxx=/usr/bin/clang++-3.9 --cc=/usr/bin/clang-3.9"
```

Comment:
I thought this change was already in 1.6?

Reply:
You are correct; my local release/1.6 was out of date. I've updated and rebased the PR on top of the current release/1.6 branch.